.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+**/node_modules
```

Comment:
- [ ] Good job adding node_modules to gitignore, however, it is currently not ignored and that is why you have so many files. To effectively ignore it, kindly remove the `**` and `/` and just type `node_modules`. It should automatically become greyed out as shown in the image below

<img width="777" alt="Screenshot 2023-06-27 at 23 20 56" src="https://github.com/MariamOsman11/Leaderboard/assets/64774936/295be4b2-0200-4f21-a6ab-695076193355">